### ISSUE TITLE
Unable to disable preview mode

### ISSUE THREAD

**Issue by Udayendu** [imzbf/md-editor-rt#112]
### Describe the issue

`preview="false"` is not working.

- Current code:
```
    <MdEditor
            hideHeader
            placeholder="Add your comment..."
            theme="light"
            language="en-US"
            preview="false"
            scrollAuto="true"
            noHighlight="true"
            modelValue={comment}
            onChange={(modelValue) => {
              handleCommentChange(modelValue);
            }}
            toolbars={[
              "bold",
              "italic",
              "-",
              "title",
              "sub",
              "sup",
              "quote",
              "unorderedList",
              "orderedList",
              "task",
              "-",
              "codeRow",
              "code",
              "-",
              "revoke",
              "next",
            ]}
          />
```

### Procedure version

4.8.1

**Comment by Udayendu** [comment]
One observation:
- if I am not using `inputBoxWitdh`, then I can see the split line:
![image](https://github.com/imzbf/md-editor-rt/assets/1809177/02b097d6-9357-4a7a-bc30-a3f16b08a2be)

- But when I am using `inputBoxWitdh="500px"`, then the split like is visible at the end.
![image](https://github.com/imzbf/md-editor-rt/assets/1809177/9214afbe-ae37-40e5-90a2-7a03a554758a)

So objective is to use this as a comment box. Hence don't want that split like to show the content preview. Hence whats the recommended way to achieve it ?

**Comment by imzbf** [comment]
in jsx, it should be `preview={false}`

**Comment by imzbf** [comment]
split line is always here, And what you see is a scroll bar

and if you want a comment box. set `preview={false}`

**Comment by Udayendu** [comment]
> split line is always here, And what you see is a scroll bar
> 
> and if you want a comment box. set `preview={false}`

I applied the `preview={false}` but still I can see both the boxes. Not able to hide the available in the right side to display the rendered value.

**Comment by imzbf** [comment]
> > split line is always here, And what you see is a scroll bar
> > and if you want a comment box. set `preview={false}`
> 
> I applied the `preview={false}` but still I can see both the boxes. Not able to hide the available in the right side to display the rendered value.

Sorry, it seems to need optimization

**Comment by imzbf** [comment]
fixed in `^4.8.2`

**Comment by Udayendu** [comment]
> fixed in `^4.8.2`

Tested today and its working as expected. Thanks for the fix !!
